bitnami/flink/templates/jobmanager/service.yml

```diff
@@ -0,0 +1,72 @@
+    {{- end }}
+  {{- if or .Values.jobmanager.service.annotations .Values.commonAnnotations }}
+  annotations:
+    {{- if .Values.jobmanager.service.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.jobmanager.service.annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.jobmanager.service.metrics.annotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.jobmanager.service.type }}
+  {{- if and (eq .Values.jobmanager.service.type "LoadBalancer") (not (empty .Values.jobmanager.service.loadBalancerIP)) }}
+  loadBalancerIP: {{ .Values.jobmanager.service.loadBalancerIP }}
+  {{- end }}
+  {{- if and (eq .Values.jobmanager.service.type "LoadBalancer") (not (empty .Values.jobmanager.service.loadBalancerSourceRanges)) }}
+  loadBalancerSourceRanges: {{ .Values.jobmanager.service.loadBalancerSourceRanges }}
+  {{- end }}
+  {{- if and .Values.jobmanager.service.clusterIP (eq .Values.jobmanager.service.type "ClusterIP") }}
+  clusterIP: {{ .Values.jobmanager.service.clusterIP }}
+  {{- end }}
+  {{- if .Values.jobmanager.service.sessionAffinity }}
+  sessionAffinity: {{ .Values.jobmanager.service.sessionAffinity }}
+  {{- end }}
+  {{- if .Values.jobmanager.service.sessionAffinityConfig }}
+  sessionAffinityConfig: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.service.sessionAffinityConfig "context" $) | nindent 4 }}
+  {{- end }}
+  {{- if or (eq .Values.jobmanager.service.type "LoadBalancer") (eq .Values.jobmanager.service.type "NodePort") }}
+  externalTrafficPolicy: {{ .Values.jobmanager.service.externalTrafficPolicy | quote }}
+  {{- end }}
+  ports:
+    - name: rpc
+      port: {{ .Values.jobmanager.service.ports.rpc }}
+      targetPort: {{ .Values.jobmanager.containerPorts.rpc }}
+      {{- if and (or (eq .Values.jobmanager.service.type "NodePort") (eq .Values.jobmanager.service.type "LoadBalancer")) (not (empty .Values.jobmanager.service.nodePorts.rpc)) }}
+      nodePort: {{ .Values.jobmanager.service.nodePorts.rpc }}
+      {{- else if eq .Values.jobmanager.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: web
+      port: {{ .Values.jobmanager.service.ports.web }}
+      targetPort: {{ .Values.jobmanager.containerPorts.web }}
+      {{- if and (or (eq .Values.jobmanager.service.type "NodePort") (eq .Values.jobmanager.service.type "LoadBalancer")) (not (empty .Values.jobmanager.service.nodePorts.web)) }}
+      nodePort: {{ .Values.jobmanager.service.nodePorts.web }}
+      {{- else if eq .Values.jobmanager.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
+    - name: blob
+      port: {{ .Values.jobmanager.service.ports.blob }}
```

Comment:
If these ports are http, please add the `http-` prefix in the name. This is for better integration with Istio